addons/html_editor/static/src/core/selection_plugin.js

```diff
@@ -147,7 +181,7 @@ export class SelectionPlugin extends Plugin {
     /**
      * @return { EditorSelection }
      */
-    getEditableSelection() {
+    getEditableSelection(deep = false) {
```

Comment:
I personally would prefer an object with the key `deep` instead (like we have for setSelection for normalize)